data_structures/linked_list/doubly_linked_list.py

```diff
@@ -33,6 +33,24 @@ def __str__(self):
             current = current.next
         return "<-->".join(str(node) for node in nodes)
 
+    def __contains__(self, value):
+        current = self.head
+        while current:
+            if current.data == value:
+                return True
+            current = current.next
+        return False
+
+    def get_head_data(self):
+        if self.head:
```

Comment:
```suggestion
        return self.head.data if self.head else None
```
Same with get_tail_data()